src/__tests__/NewButton.test.tsx

```diff
@@ -63,7 +63,7 @@ describe('Button', () => {
     expect(render(<Button variant="danger">Danger</Button>)).toHaveStyleRule('background-color', '#f6f8fa')
   })
   it('styles icon only button to make it a square', () => {
-    const IconOnlyButton = render(<Button icon={SearchIcon}>Search icon only button</Button>)
+    const IconOnlyButton = render(<IconButton icon={SearchIcon} iconLabel="Search icon only button" />)
```

Comment:
One for another time, but wanted to put it on your radar. Can we move away from Enzyme to use the render from testing library instead? It's already imported here as `HTMLRender`. On the disabled test for example, we can instead look for `disabled` html attributes instead of testing the implementation props. What do you think?

Reply:
Good idea! Its much nicer to use testing library. Rewrote the tests in that.